webpack.config.js

```diff
@@ -57,19 +25,42 @@ module.exports = {
 					loader: 'babel-loader',
 				},
 			},
-			{
-				test: /style\.s?css$/,
```

Comment:
It looks like this plugin doesn't use `style.css` or `style.scss`. I couldn't find a reference to it, though maybe I missed something.